code/game/objects/items/hand_items.dm

```diff
@@ -365,6 +365,9 @@
 	if (user.body_position != LYING_DOWN)
 		return
 	var/obj/item/item_to_strip = target_human.shoes
+	if(target_human.shoes.chained && target_human == user)
```

Comment:
Chained should be a separate check and we should block people from stealing their own shoes period, I don't see any reason for someone to be able to 'steal' their own shoes.

Reply:
for some reason the chained check is giving me nothing but issues. what should the separate check for chained be doing, anyway - it doesn't seem to adjust strip delay and the issue being addressed here is just about people taking chained shoes off by themselves, which is addressed by the self check

Reply:
Actually my comment was pretty stupid, people should be able to strip off the chained shoes